mitodl/mitxonline
# 3005: adding a banner snippet in the header that appears on every page of the site

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/3005

### Description (What does it do?)
Adds a banner to the website.

### Screenshots (if appropriate):
![Screenshot 2024-03-27 at 3 57 14 PM](https://github.com/mitodl/mitxonline/assets/8311573/5f4cd505-25a7-4d9e-a569-ba7609f72449)

### How can this be tested?

1. Navigate to http://mitxonline.odl.local:8013/cms/
2. Click "Snippets".
3. Click "Site banners".
4. Click "Add site banner".
5. Write something in the text box and save.
6. Visit the MIT application's homepage http://mitxonline.odl.local:8013/ and verify that banner is showing.
7. Verify that banner can be closed and does not re-appear when navigating to other pages.

The button used for closing the banner is a material icon which aligns with xPro and what is used throughout MITx Online.  It does differ from the mock-ups.
